### DISCUSSION TITLE
Can We set password for Skywalking Controller?

### DISCUSSION THREAD

**Discussion by surya-muthuraj0** [apache/skywalking#9801] [Q&A]
How can we set password to the skywalking environment.

**Comment by wu-sheng** [comment]
What is SkyWalking controller? We don't have that concept.

**Reply by surya-muthuraj0** [reply]
Skywalking UI

**Reply by wu-sheng** [reply]
Read doc please, https://skywalking.apache.org/docs/main/next/en/ui/readme/#login-and-authentication
